docs/src/modules/components/AppFooter.js

```diff
@@ -161,18 +171,18 @@ function AppFooter(props) {
                     Contact Us
                   </Link>
                 </li>
-                <li className={classes.careers}>
+                <CareersLi>
```

Comment:
```suggestion
                <Box sx={{ display: 'flex' }} component="li">
```

For one css rule, so far we used the `sx` prop. It helps with avoiding switching context. But as this is not some demo that is publicly visible, this should be fine as well, as the migration is simpler.